acme/acmego/main.go

```diff
@@ -21,7 +21,6 @@
 // The other known extensions and formatters are:
 //
 //	.rs - rustfmt
-//
```

Comment:
This line got deleted, ironically, by acmego. I need to add a comment about .c and .h here.